tests/wicked/basic/ref/t08_setup_second_card.pm

```diff
@@ -22,11 +22,15 @@ use lockapi;
 sub run {
     my ($self, $ctx) = @_;
     record_info('Info', 'Set up a second card');
-    assert_script_run(sprintf("ip a a %s/15 dev %s", $self->get_ip(type => 'second_card'), $ctx->iface()));
+    assert_script_run(sprintf("ip a a %s/24 dev %s", $self->get_ip(type => 'dhcp_2nic'),   $ctx->iface()));
```

Comment:
one of the lines suppose to contain `iface2` , right ?

Reply:
there is no iface2 in REF. We are using IP alias here

Reply:
ok I get it , but just FYI there **is** second interface on REF check any REF job settings